core/src/banking_stage/transaction_scheduler/prio_graph_scheduler.rs

```diff
@@ -233,10 +229,13 @@ impl<Tx: TransactionWithMeta> Scheduler<Tx> for PrioGraphScheduler<Tx> {
                 );
 
                 match maybe_schedule_info {
-                    Err(TransactionSchedulingError::UnschedulableConflicts)
-                    | Err(TransactionSchedulingError::UnschedulableThread) => {
+                    Err(TransactionSchedulingError::UnschedulableConflicts) => {
+                        num_unschedulable_conflicts += 1;
```

Comment:
nitpick: use original `saturating_add_assign!` for consistency?

Reply:
ah i just changed it to be consistent with the greedy scheduler's accumulation

Reply:
Yea, I recall we talked about this when reviewing greedy scheduler, and agreed `+= 1` here is proper. What I meant to say is there are still few `saturating_add_assign!` in same function here, which gave me a pause when reading, wondering why some counters are `+=1` others `saturating_add_assign`.

Reply:
Good point; I'll go through in a follow-up PR to make the two branches more consistent in their accumulation